docs/DeveloperGuide.md

```diff
@@ -944,6 +944,138 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
 - `remark 100000 r/Birthday on 31 July.`.
 - `remark 1 r/Birthday on 31 July r/`.
 
+### Scheduling a meeting with a person
+
+#### Scheduling a meeting with a person successfully
+##### The time input of this test must be after current date time.
```

Comment:
I think this should not be a header? I think we should use a text either in a `<div>` block or simply a bolded text?